src/libs/actions/App.js

```diff
@@ -188,11 +188,16 @@ function setUpPoliciesAndNavigate(session) {
     const url = new URL(currentUrl);
     const exitTo = url.searchParams.get('exitTo');
 
+    // Approved Accountants and Guides can enter a flow where they make a workspace for other users,
+    // and those are passed as a search parameter when using transition links
+    const ownerEmail = url.searchParams.get('ownerEmail');
+    const makeMeAdmin = url.searchParams.get('makeMeAdmin');
```

Comment:
So if alex was my guide and created the policy for me:
* Alex is hitting this URL via his new dot
* `ownerEmail` is andrew@email.com
* `makeMeAdmin` is `true`

Is that correct?

Reply:
Correct, Alex would be hitting this through a transition link generated by OldDot